### PR TITLE
Devops/872/update pa11y config

### PR DESCRIPTION
**Pull request by riatzukiza**
### Summary of Changes

Added the rest of the frontend views to the pa11y config file
Added artifacts to circle ci

### How to Test

1. initiate the circle CI test sequence
2. when ci job is done, check the artifacts and verify that the following files are present:

+ tdrs-frontend/pa11y-screenshots/data-files.png
+ tdrs-frontend/pa11y-screenshots/edit-profile.png
+ tdrs-frontend/pa11y-screenshots/homepage.png
+ tdrs-frontend/pa11y-screenshots/login.png
+ tdrs-frontend/pa11y-screenshots/request.png

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [x] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [x] Check against the criteria:

As facilitator/product manager, @kniz-raft will decide if ACs are met from Raft's perspective.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

+ Are all areas of code introduced in this PR meaningfully tested?
  + [x] If this PR introduces backend code changes, are they meaningfully tested?
  + [x] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
  + [x] Backend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

+ [x] Are backend code style checks passing on CircleCI?
+ [x] Are frontend code style checks passing on CircleCI?
+ [x] Does this PR change any linting or CI settings?

### Deliverable 4: Accessible

> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*

> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*

+ [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful build with a single command*

> NOTE: until we have a proper staging environment this may not be satisfiable prior to merging

+ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

+ [x] If this PR introduces backend code, is that code documented both inline and overall?
+ [x] If this PR introduces frontend code, is that code documented both inline and overall?
+ [x] If this PR introduces dependencies, are their licenses documented?

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

+ [x] Does the OWASP Scan pass on CircleCI?
+ [x] Do manual code review and manual testing detect any security issues?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -255,7 +255,7 @@ jobs:
           command: "cd tdrs-frontend; yarn lint"
       - run:
           name: "Run Pa11y Accessibility Tests"
-          command: "cd tdrs-frontend; yarn test:accessibility"
+          command: "cd tdrs-frontend; mkdir pa11y-screenshots/; yarn test:accessibility"
```
Comment (riatzukiza) [inline .circleci/config.yml:258]: With out creating the folder, circle CI failed to find the files.

**Review by jorgegonzalez — APPROVED** [review]
LGTM 👍

File: .circleci/config.yml [hunk 2]
```diff
@@ -276,6 +276,20 @@ jobs:
...
       - store_artifacts:
           path: tdrs-frontend/reports/owasp_report.html
+      - store_artifacts:
+          path: tdrs-frontend/pa11y-screenshots/homepage.png
+
+      - store_artifacts:
+          path: tdrs-frontend/pa11y-screenshots/edit-profile.png
+
+      - store_artifacts:
+          path: tdrs-frontend/pa11y-screenshots/request.png
```
Comment (ADPennington) [inline .circleci/config.yml:286]: @riatzukiza is there a meaningful distinction between this and the homepage?

Comment (riatzukiza) [reply .circleci/config.yml:286]: yes, this is the page the user is directed to after they have entered their name and stt in /edit-profile. This PR doesn't login to the app, so all of the endpoints that require authentication show the home page with a message telling the user to login. I made a new issue to set up pa11y to login to the app using a puppeteer browser instance.

**Review by ADPennington — APPROVED** [review]
thanks @riatzukiza! 👍🏾 @abottoms-coder this is ready to merge. 

---
### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [x] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [x] Check against the criteria:
  + [x] verify that the following files are present:
       - tdrs-frontend/pa11y-screenshots/data-files.png
       - tdrs-frontend/pa11y-screenshots/edit-profile.png
       - tdrs-frontend/pa11y-screenshots/homepage.png
       -  tdrs-frontend/pa11y-screenshots/login.png
       - tdrs-frontend/pa11y-screenshots/request.png
       
As Product Owner, @lfrohlich will decide if ACs are met.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

+ Are all areas of code introduced in this PR meaningfully tested?
  + [x] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
  + [x] Backend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

+ [x] Are backend code style checks passing on CircleCI?
+ [x] Are frontend code style checks passing on CircleCI?

### Deliverable 4: Accessible

_n/a_

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful build with a single command*

+ [x] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

+ [x] If this PR introduces frontend code, is that code documented both inline and overall?
_noting here for the reference that dev has created a follow-on ticket #1044  to capture artifacts for screens that require authentication_
### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

+ [x] Does the OWASP Scan pass on CircleCI?
+ [x] Do manual code review and manual testing detect any security issues?
_no new issues detected_

**Pull request merged**